addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -60,8 +67,81 @@ Thing exec:command:myscript [command="php ./configurations/scripts/script.php %2
 **demo.items**
 
 ```
-String APCRaw "[%s]" (All) {channel="exec:command:apc:output"}
-Switch APCRunning { channel="exec:command:apc:run"}
-Number APCExitValue {channel="exec:command:apc:exit"}
-DateTime APCLastExecution {channel="exec:command:apc:lastexecution"}
+
+String APCRaw "[%s]" (All) {channel="exec:command:apc:output"} 
+String APCRunning { channel="exec:command:apc:run"}
+String APCExitValue {channel="exec:command:apc:exit"}
+String APCLastExecution {channel="exec:command:apc:lastexecution"}
+```
+
+## Full Example
+
+Following is an example how to set up an exec Thing, debug it with a rule and set the returned string to an Number Item. 
+
+**For this to work also the openhab-transformation-regex has to be installed**
```

Comment:
the openHAB RegEx Transformation

Reply:
But it is called like this in Karaf when i try to install it!

Reply:
Yes but that's just the technical package name. In Paper UI or in documentation it has a human readable name. In the last sentence I used the name "Paper UI" not its feature name `openhab-ui-paper`, that would be unintuitive.

The Console (karaf) is just one tool to manage openHAB and documentation should stay general and end user oriented. The end user needn't know the feature name, if he/she wants, it's easily found out via `feature:list` in said console.